components/prism-stylus.js

```diff
@@ -78,13 +81,18 @@
 		'property-declaration': {
 			pattern: /((?:^|\{)([ \t]*))(?:[\w-]|\{[^}\r\n]+\})+(?:\s*:\s*|[ \t]+)[^{\r\n]*(?:;|[^{\r\n,](?=$)(?!(?:\r?\n|\r)(?:\{|\2[ \t]+)))/m,
 			lookbehind: true,
+			greedy: true,
 			inside: {
 				'property': {
 					pattern: /^[^\s:]+/,
 					inside: {
 						'interpolation': inside.interpolation
 					}
 				},
+				'comment': {
```

Comment:
It seems weird to me that you have to add that here? Shouldn't this be part of `inside` (the variable) itself? I mean, can there be no comments inside `atrule-declaration` tokens?
If you make `comment` part of `inside`, please replace the `comment` of the stylus language with `inside['comment']` (similar to `func` and `string`).

Also, since `comment` isn't the first token, it should probably be greedy.